### DISCUSSION TITLE
使用PaddleServing部署模型时，怎么指定预测模型的各类参数

### DISCUSSION THREAD

**Discussion by HHaidai** [PaddlePaddle/PaddleOCR#13929] [Q&A]
![image](https://github.com/user-attachments/assets/7a526436-3e2f-4f39-a2df-322333a4cf6d)
这些参数我该在哪里指定呢？

**Comment by HHaidai** [comment]
db模型的部分参数竟然是web_service.py里面直接写死的，官方是不是应该改一改.
![image](https://github.com/user-attachments/assets/ccb6281e-f7df-4a35-83b5-b4f030276e6a)

**Reply by HHaidai** [reply]
使用pdserving部署的模型的预测参数改其他地方没用，只能在这里面改......

**Comment by GreatV** [comment]
欢迎提交PR改进此功能
